app/views/curation_concerns/single_use_links_viewer/show.html.erb

```diff
@@ -0,0 +1,21 @@
+<div >
+  <h1 class="lower"><%= @asset %></h1>
+    <h2 class="non lower">Actions</h2>
+    <p>
+      <%= link_to "Download (can only be used once)", @download_link %>
+    </p>
+  <h2> Descriptions:</h2>
+
+  <div class="well active" id="descriptions_display">
+    <table class="table table-striped">
+      <tbody>
+      <% present_terms(@presenter) do |r, term| %>
```

Comment:
This bit here at line 12 is what doesn't work because of the missing `present_terms` [helper method](https://github.com/projecthydra/sufia/blob/a064489d3cf4d228abf978d7126644e2c653aa5e/app/helpers/generic_file_helper.rb#L7), which calls the missing `PresenterRenderer` [class](https://github.com/projecthydra/sufia/blob/a064489d3cf4d228abf978d7126644e2c653aa5e/app/presenters/sufia/presenter_renderer.rb).